modules/representation_management/app/models/representation_management/form_2122a_data.rb

```diff
@@ -23,5 +34,21 @@ class Form2122aData < RepresentationManagement::Form2122Base
     validates :veteran_service_branch,
               inclusion: { in: VETERAN_SERVICE_BRANCHES },
               if: -> { veteran_service_branch.present? }
+
+    def representative_field_truncated(field)
+      if representative.public_send(field)
+        representative.public_send(field)[0..TRUNCATION_LIMITS[field] - 1]
+      else
+        ''
+      end
+    end
```

Comment:
What do you think about something like this:

```ruby
    def representative_field_truncated(field)
      result = representative.public_send(field)
      max_index = TRUNCATION_LIMITS[field] - 1 

      raise StandardError. "#{field} does not have a truncation limit defined" unless max_index

      if result.present?
        result[0..max_index]
      else
        ''
      end
    end
```

Reply:
This looks good!

It's a done deal with an additional test to make sure `StandardError` is raised.